### DISCUSSION TITLE
RenderAnim Zip Option Times Out or Never Completes

### DISCUSSION THREAD

**Discussion by andrejhronco** [cables-gl/cables_docs#226] [! Welcome - readme first!]
Hi,
Experimenting with different settings for `RenderAnim` and longer record times with higher the FPS cause the `Zip Multiple Files` option to take extremely long or never complete. For example; 30sec at 60FPS takes 8 minutes to complete the zipping procedure and download the file. Longer record times just don't work. Are there some other settings for JSZip that might help fix this? Would also be good to use the callback option for `generateAsync` to show progress time out of a port.

when it fails, there's an error in the console `RangeError: Array buffer allocation failed`
![image](https://user-images.githubusercontent.com/1925905/163735294-1dabeea8-98a7-4899-b1f3-39bf5d8f832a.png)

45 seconds at 60FPS seems to be the limit on an M1 16GB

**Comment by steam0r** [comment]
hey,

there is nothing we can do about that sadly, thats basically your browser running out of memory. there is different restrictions on different browsers on how much memory a webpage is allowed to load...you might have better luck in other browsers...or not using the zip-option at all and allowing your browser to download multiple files. sorry...

**Reply by andrejhronco** [reply]
yeah, makes sense. just thought 16GB would leave plenty of headroom. Thanks
